bin/asciibinder

```diff
@@ -17,7 +18,15 @@ def call_generate(distro,page=nil)
   end
 end
 
-SUB_COMMANDS = %w{help build watch package clean create}
+def repo_check(repo_dir)
+  ['.git','_build_cfg.yml','_distro_map.yml','_templates'].each do |file|
+    if not File.exist?(File.join(repo_dir, file))
```

Comment:
I picked up this pattern from Clayton; perhaps it is a Rails-ism but not general Ruby practice? In any case I am happy to change it. What is the preferred pattern for this?